packages/wdio-browserstack-service/src/cucumber-types.ts

```diff
@@ -0,0 +1,197 @@
+// mimic types from @cucumber/@cucumber such that users don't install that dependency when not needed
+
+export interface ITestCaseHookParameter {
+    gherkinDocument: GherkinDocument;
+    pickle: Pickle;
+    result?: TestStepResult;
+    willBeRetried?: boolean;
+    testCaseStartedId: string;
+}
+
+export declare class Duration {
+    seconds: number
+    nanos: number
+}
+
+export declare class GherkinDocument {
+    uri?: string
+    feature?: Feature
+    comments: readonly Comment[]
+}
+
+export declare class Background {
+    location: Location
+    keyword: string
+    name: string
+    description: string
+    steps: readonly Step[]
+    id: string
+}
+
```

Comment:
missing:
```ts
export declare class Comment {
    location: Location
    text: string
}
```

Reply:
Added. Nice catch. Fortunately, the type getting used was some inbuilt type that's why no error popped up either while building or while testing.